Cargo.lock

```diff
@@ -5404,6 +5405,9 @@ name = "lazy_static"
 version = "1.5.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "bbd2bcb4c963f2ddae06a2efc7e9f3591312473c50c6685e1f298068316e66fe"
+dependencies = [
+ "spin 0.9.8",
+]
```

Comment:
same: do we need this?

Reply:
it comes from lazy static from the substra-bn